app/routes.js

```diff
@@ -37,3 +37,15 @@ require('./routes/withdraw-offer')(router)
 
 // examples
 require('./routes/examples')(router)
+
+// viewing session data
+router.get('*/manage-prototype/view-data', function(req, res){
+
+    querystring = '';
+    for ( var key in req.session.data )
+    {
+        querystring += key +'=' + req.session.data[key] + '&';
+    }
+
+    res.render('manage-prototype/view-data', { data: JSON.stringify( req.session, null, 2), querystring: querystring } );
+  })
```

Comment:
```suggestion
router.get('*/manage-prototype/view-data', (req, res) => {
  let querystring = ''
  for (let key in req.session.data) {
    querystring += key + '=' + req.session.data[key] + '&'
  }
  
  res.render('manage-prototype/view-data', { 
    data: JSON.stringify(req.session, null, 2), 
    querystring
  })
})
```

An alternative to this is to use `{{ data | log }}` in your main layout view, for example: https://github.com/DFE-Digital/find-teacher-training-prototype/blob/4e85bd9fafbe22726b310f857b3983bc4fec2e64/app/views/_layouts/default.njk#L82

Reply:
Thanks @simonwhatley.